qa/integration-tests/src/test/java/io/camunda/zeebe/it/clustering/topology/dynamic/DynamicClusterTopologyServiceTest.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.it.clustering.topology.dynamic;
+
+import io.atomix.cluster.MemberId;
+import io.camunda.zeebe.qa.util.cluster.TestCluster;
+import io.camunda.zeebe.qa.util.cluster.TestStandaloneBroker;
+import io.camunda.zeebe.qa.util.junit.ZeebeIntegration;
+import io.camunda.zeebe.qa.util.junit.ZeebeIntegration.TestZeebe;
+import java.time.Duration;
+import org.junit.jupiter.api.Test;
+
+@ZeebeIntegration
+final class DynamicClusterTopologyServiceTest {
+  @TestZeebe
+  private final TestCluster cluster =
```

Comment:
:heart_eyes:

Reply:
Be aware that by default the `@TestZeebe` annotation is already automatically starting the cluster and awaiting the topology, so you don't need to await it in the test too :sweat_smile:

Reply:
Nice. I will do that otherwise test method will be empty smile: